Game engines websites (#5126)
github/explore: topics/pyxel/index.md

---
display_name: Pyxel

> short_description: Internet Archive is a website for a digital collection run by the archive.org group.

Reviewer: ```suggestion
short_description: Pyxel is a retro game engine for python.
```

> aliases: wayback-machine

Reviewer: ```suggestion
```

topic: pyxel
---

> Internet Archive is a website for a digital collection run by the archive.org group, also responsible for the Wayback Machine software.

Reviewer: ```suggestion
Pyxel is a small open source game engine written in Python for building games evocative of retro gaming consoles.
```

Reviewer: Somehow this got confused with the internet archive descriptions.
